src/TableDefinition.php

```diff
@@ -16,8 +16,9 @@ public function __construct(
         public bool $trailing = false,
         public array $formatOptions = [],
         public ?string $when = null,
-    ) {
-    }
+        public bool $csv = false,
+        public bool $withEmbedded = false,
```

Comment:
Is it better to set `$withEmbedded = true` by default when using CSV?

Reply:
I actually thought of setting it to true by default. Since you noticed it too, that should be the assurance I need. Will update it.

Reply:
Looks good, please fix the failing test. Thanks!